src/openms/include/OpenMS/MATH/STATISTICS/PosteriorErrorProbabilityModel.h

```diff
@@ -125,13 +150,13 @@ namespace OpenMS
       }
 
       ///computes the gaussian density at position x with parameters params.
-      double getGauss(double x, const GaussFitter::GaussFitResult & params)
+      static double getGauss_(double x, const GaussFitter::GaussFitResult & params)
       {
         return params.A * exp(-1.0 * pow(x - params.x0, 2) / (2 * pow(params.sigma, 2)));
```

Comment:
note: I remember `pow(x, 2)` being very slow compared to `x*x`.

Reply:
I just tested it on godbolt and both produce the same asm code - except for debug mode (then x*x is faster).